src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/ReferenceSearchParam.sql

```diff
@@ -42,3 +42,15 @@ WHERE IsHistory = 0
 WITH (DATA_COMPRESSION = PAGE)
 ON PartitionScheme_ResourceTypeId(ResourceTypeId)
```

Comment:
Please remove this index

Reply:
I removed the index.  Should I also put a DROP INDEX statement in the 68.diff.sql ? To remove the existing index on upgrade?  Or should we leave it there?

Reply:
Please DROP